spec/openapi.yml

```diff
@@ -884,6 +890,26 @@ components:
         description: Sorts the results of your query by indicated direction `asc` or `desc`.
       required: false
 
+    pattern:
+      name: pattern
+      in: query
+      schema:
+        default: '%'
+        type: string
+        example: etl
+        description: Filters the results of your query by indicated pattern. Matches any substring
+      required: false
+
+    orderBy:
+      name: orderBy
+      in: query
+      schema:
+        type: string
+        default: name
+        example: name
+        description: Orders the results of your query by indicated `sortDirection` by any column on the row.
```

Comment:
Could we mention few other examples of ordering? Otherwise users may find it difficult to use them.

Reply:
Yeah, I can add some examples from the shared column names between jobs and datasets.